ADD-ONS-CHANGELOG.md

```diff
@@ -0,0 +1,59 @@
+# React Add-Ons Change Log
```

Comment:
Maybe let’s just keep it in the main changelog? It seems like they could fit in fine if grouped under React Addons for each release (see like we did before).